Translations/cs.lproj/Localizable.strings

```diff
@@ -202,21 +202,21 @@ Apple needs to have MAS translated in this language so this strings can be used
 The language used in MAS/AppleID will be used in Keka's tip strings and may not match Keka's localization
 Note that the short string is limited to 30 characters and the long string is limited to 45 characters.
 */
-"Monthly sip of coffee" = "Monthly sip of coffee";
-"Gift Keka a sip of coffee each month" = "Gift Keka a sip of coffee each month";
-"Monthly coffee" = "Monthly coffee";
-"Gift Keka a coffee each month" = "Gift Keka a coffee each month";
-"Monthly pair of coffees" = "Monthly pair of coffees";
-"Gift Keka a couple coffees each month" = "Gift Keka a couple coffees each month";
-"Monthly breakfast" = "Monthly breakfast";
-"Gift Keka a breakfast each month" = "Gift Keka a breakfast each month";
-"Tip a coffee" = "Tip a coffee";
-"Tip Keka a coffee so it can stay awake" = "Tip Keka a coffee so it can stay awake";
-"Tip a breakfast" = "Tip a breakfast";
-"Tip Keka a breakfast so it can start fresh" = "Tip Keka a breakfast so it can start fresh";
-"Tip a lunch" = "Tip a lunch";
-"Tip Keka a lunch and feed it" = "Tip Keka a lunch and feed it";
-"Tip a dinner" = "Tip a dinner";
-"Tip Keka a dinner so it keeps rolling" = "Tip Keka a dinner so it keeps rolling";
-"Tip a bunch of food" = "Tip a bunch of food";
-"Tip Keka a new bunch of food to keep rolling" = "Tip Keka a new bunch of food to keep rolling";
+"Monthly sip of coffee" = "Pár zrnek kávy";
+"Gift Keka a sip of coffee each month" = "Keka si pochutná na pár zrncích kávy";
+"Monthly coffee" = "Kafe na pár dní";
+"Gift Keka a coffee each month" = "Keka bude mít energii na pár dní";
+"Monthly pair of coffees" = "Kafe na celý měsíc";
+"Gift Keka a couple coffees each month" = "Keka bude mít energii na celý měsíc";
+"Monthly breakfast" = "Snídaně na měsíc";
+"Gift Keka a breakfast each month" = "Keka si pochutná na dobré snídani";
+"Tip a coffee" = "Jedno kafe";
+"Tip Keka a coffee so it can stay awake" = "Kupte Keka kávu, aby byla plná energie";
+"Tip a breakfast" = "Jedna snídaně";
+"Tip Keka a breakfast so it can start fresh" = "Kupte Keka snídani, aby se jí dobře vstávalo";
+"Tip a lunch" = "Jeden oběd";
+"Tip Keka a lunch and feed it" = "Kupte Keka oběd, aby byla zdravá";
+"Tip a dinner" = "Jedna večeře";
+"Tip Keka a dinner so it keeps rolling" = "Kupte Keka večeři, aby se jí dobře spalo";
+"Tip a bunch of food" = "Stůl plný jídla";
+"Tip Keka a new bunch of food to keep rolling" = "Pozor, aby neztloustla!";
```

Comment:
🤣 Although I've laughed a lot when translating this one, I'm not sure Apple reviewers will approve it since it does not describe the tip. But lets try 😉

Reply:
I had to get really creative because if I translated the originals exactly it would sound really awkward, as the word for "tip" is used exclusively in the context of restaurant tipping 😅 I hope they do, when I interviewed with Apple, I had to get creative like this when translating some their stuff, so let's hope they didn't change anything since then... 😳